build.sh

```diff
@@ -77,4 +77,4 @@ export PATH="$DOTNET_INSTALL_DIR:$PATH"
 # Disable first run since we want to control all package sources
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-dotnet msbuild $REPOROOT/build/build.proj /m:1 /nologo /p:Configuration=$CONFIGURATION /p:Platform="$PLATFORM" "${args[@]}"
+dotnet build $REPOROOT/build/build.proj /m:1 /nologo /flp:logfile=project.log;verbosity=diagnostic /p:Configuration=$CONFIGURATION /p:Platform="$PLATFORM" "${args[@]}"
```

Comment:
Why did the build command have to change?  Was this local only stuff to see what was going on?  Can we undo this line?